config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -1404,14 +1404,87 @@ spec:
+                                  / Secret
+                                type: string
+                              certKey:
+                                description: Certificate private key file name within
+                                  the ConfigMap / Secret. Omit when the key is not
+                                  necessary.
+                                type: string
+                              name:
+                                description: Name of the ConfigMap or Secret containing
+                                  certificates
+                                type: string
+                              type:
+                                description: 'Reference type: configmap or secret'
+                                enum:
+                                - configmap
+                                - secret
+                                type: string
+                            type: object
+                          enable:
+                            default: false
+                            description: Enable TLS
+                            type: boolean
+                          insecureSkipVerify:
+                            default: false
+                            description: Skip client-side verification of the server
+                              certificate
+                            type: boolean
+                          userCert:
+                            description: User certificate reference
+                            properties:
+                              certFile:
+                                description: Certificate file name within the ConfigMap
+                                  / Secret
+                                type: string
+                              certKey:
+                                description: Certificate private key file name within
+                                  the ConfigMap / Secret. Omit when the key is not
+                                  necessary.
+                                type: string
+                              name:
+                                description: Name of the ConfigMap or Secret containing
+                                  certificates
+                                type: string
+                              type:
+                                description: 'Reference type: configmap or secret'
+                                enum:
+                                - configmap
+                                - secret
+                                type: string
+                            type: object
+                        type: object
+                      port:
+                        default: 9090
+                        description: the prometheus HTTP port
+                        format: int32
+                        maximum: 65535
+                        minimum: 1
+                        type: integer
+                      tlsType:
```

Comment:
What's the difference between `tlsType: DISABLED` and `enable: false` ?

Reply:
Thanks, I changed it and manualtls only contain cert configuration now.